### PR TITLE
added release branches to renovate baseBranches config

### PR DESCRIPTION
**Pull request by vardhaman22**
https://github.com/rancher/cis-operator/issues/360 and https://github.com/rancher/cis-operator/issues/361
- added regex for branches having prefix **release/** in renovate config so that renovate raises PR for all the active base branches.

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "github>rancher/renovate-config#release"
   ],
   "baseBranches": [
-    "main"
+    "main",
+     "/^release\\/.*/"
```
Comment (pjbgf) [inline .github/renovate.json:7]: It's OK for now, but we will have to change this once one of our release branches goes EOL.

**Review by pjbgf — APPROVED** [review]

**Review by pjbgf — CHANGES_REQUESTED** [review]
Actually, I think we should define the specific branches, and set at least the Kubernetes versions we aim to go for on each release line.

File: .github/renovate.json (hunk 1 above)
Comment (vardhaman22) [reply .github/renovate.json]: done

File: .github/renovate.json [hunk 2]
```diff
@@ -3,7 +3,59 @@
...
+        "k8s.io/client-go"
+      ],
+      "allowedVersions": "<1.32.0"
+    },
+    {
+      "matchBaseBranches": ["release/v1.2"],
+      "matchDepNames": [
+        "github.com/rancher/lasso"
+      ],
+      "allowedVersions": "<v0.0.0-20240924233157-8f384efc8813"
```
Comment (pjbgf) [inline .github/renovate.json:31]: Not sure how we are going to manage this, due to the overlap with some k8s dependencies and the lack of Lasso's branching per release line.

However, this is a good start.

File: .github/renovate.json [hunk 3]
```diff
@@ -3,7 +3,59 @@
...
   ],
   "baseBranches": [
-    "main"
+    "main", 
+    "release/v1.1", 
+    "release/v1.2", 
+    "release/v1.3"
   ],
-  "prHourlyLimit": 2
+  "prHourlyLimit": 2,
```
Comment (pjbgf) [inline .github/renovate.json:11]: We may need to increase this given the different base branches. Let's revisit this as we go.

**Review by pjbgf — DISMISSED** [review]

**Review by pjbgf — APPROVED** [review]

File: .github/renovate.json (hunk 3 above)
Comment (vardhaman22) [reply .github/renovate.json:11]: sure

File: .github/renovate.json (hunk 2 above)
Comment (vardhaman22) [reply .github/renovate.json:40]: okay

**Pull request merged**